src/main/git-elegant-feature

```diff
@@ -2,7 +2,17 @@
 set -e
 
 default() {
+    local uncommited_changes=false
     _validate "$1" "Feature name"
+    local status="git status | grep 'Changes to be commited: || Changes not staged for commit:'"
```

Comment:
There is a bug: git elegant will stash even there are no changes. Please create unit test, then fix.

Log:
```bash
~/data/elegant-git git status
On branch stash3
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	src/main/test

nothing added to commit but untracked files present (use "git add" to track)
~/data/elegant-git git status | grep 'Changes to be commited: || Changes not staged for commit:'
~/data/elegant-git status="git status | grep 'Changes to be commited: || Changes not staged for commit:'"
~/data/elegant-git if [ -n $(eval "$status") ]; then echo "ff"; fi
ff
```